src/components/star.js

```diff
@@ -0,0 +1,5 @@
+function Star({ filled }) {
+    return <span className={filled ? "star filled" : "star"}>{filled ? '★' : '☆'}</span>;
```

Comment:
Not a fan about using special characters on HTML as it could break HTML according to the computer if the character does not exist.
Please use only css to be able to show the filled start and the non-filled star